src/components/PeriodicTable.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useRef } from "react"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['useEffect' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)